src/lualib/universal/Modulo50.ts

```diff
@@ -0,0 +1,5 @@
+/** @noSelfInFile */
+
+export function __TS__Modulo50(_a: number, _b: number) {
+    throw "Using a 5.0 shim for a non-5.0 target, what?";
```

Comment:
This file probably just should not exist

Reply:
Done, should have thought of it first.